plugins/modules/vm_boot_devices.py

```diff
@@ -153,22 +154,56 @@
 RETURN = r"""
 record:
   description:
-    - VM's device that we're assigning the boot order to, which can be either disks from the API endpoint C(/VirDomainBlockDevices), or
-      nics from the API endpoint C(/VirDomainNetDevices).
+    - VM's device that we're assigning the boot order to, which can be either disks from the API endpoint C(/VirDomainBlockDevices),
+      or nics from the API endpoint C(/VirDomainNetDevices).
   returned: success
   type: dict
-  sample:
-    cache_mode: none
-    disable_snapshotting: false
-    disk_slot: 2
-    mount_points: []
-    name: ""
-    read_only: false
-    size: 10737418240
-    tiering_priority_factor: 8
-    type: virtio_disk
-    uuid: d48847d0-91b1-4edf-ab28-3be864494711
-    vm_uuid: 183c5d7c-1e2e-4871-84e8-9ef35bfda5ca
+  contains:
+    cache_mode:
+      description: The cache mode the block device will use
+      type: str
+      sample: WRITEBACK
```

Comment:
I run `examples/vm_info.yml` , I got
```
      boot_devices:
      - cache_mode: writethrough
```

Reply:
Thanks for noticing, I already fixed it (and also in some other modules)